sc2rf.py

```diff
@@ -32,7 +33,9 @@
     '': (29534, 99999), # probably nothing, but need something to mark the end of N
 }
 
+
 class Interval:
+    """ A closed interval of non-negative integers, e.g., 27 or 28-30 """
```

Comment:
This class can also represent open intervals, as stated in the argument description: "An Interval can be a single number ("3"), a closed interval ("2-5" ) or an open one ("4-" or "-7"). The limits are inclusive. Only positive numbers are supported."

(other than this, your comments on my code are spot on)

Reply:
whoops. I'll update on my end, I think it should propagate to this PR

Reply:
BTW, when I checked if my code _actually_ supports open intervals, I found a bug (which only affected the interval for the `-s` arg) which I just fixed.